### PR TITLE
Updated screenshots

### LINKED ISSUE #249: bug: screenshots broken on main page of this repository
The [user interface section of this repository's `README.md`](https://github.com/amilajack/freesound-player#ui) currently has broken screenshots. We should fix this by changing lines 9 and 12 of [`freesound-player/README.md`](https://github.com/amilajack/freesound-player/blob/master/README.md).

By the way, we can remove the
### Day Theme
header from `README.md` since we're only showing the appearance during one time of day now.

Screen Shot 2020-12-01 at 12 56 35 PM

### PR DESCRIPTION
**Pull request by akshara-sun**
closes #249

### CODE REVIEW

**Comment by amilajack** [comment]
This doesn't fix the issue. It just deletes the screenshots. we need to restore the original images that were here. You can do this by going back in time (by browsing the project's git history). Find the images in the previous commit before the dropdown/modal change. then copy and paste those images to the `screenshots` dir.

**Review by amilajack — CHANGES_REQUESTED** [review]
This doesn't fix the issue. It just deletes the screenshots. we need to restore the original images that were here. You can do this by going back in time (by browsing the project's git history). Find the images in the previous commit before the dropdown/modal change. then copy and paste those images to the screenshots dir.

**Comment by akshara-sun** [comment]
> This doesn't fix the issue. It just deletes the screenshots. we need to restore the original images that were here. You can do this by going back in time (by browsing the project's git history). Find the images in the previous commit before the dropdown/modal change. then copy and paste those images to the `screenshots` dir.

@amilajack I didn't do go back in time in the commit history because we added a banner to the home page so I deleted the old screenshots and took new ones and added them.

**Comment by amilajack** [comment]
I see. Looks like i didnt review carefully enough. the files are in binary after all lol

**Review by amilajack — APPROVED** [review]

**Comment by amilajack** [comment]
Screen Shot 2020-12-02 at 12 31 25 PM
merged and screenshots still broken

**Comment by akshara-sun** [comment]
> Screen Shot 2020-12-02 at 12 31 25 PM
> 
> merged and screenshots still broken

I wonder why...let me try again. Thanks for double checking.
